lib/alchemy/upgrader/tasks/ingredients_migrator.rb

```diff
@@ -13,12 +13,18 @@ def create_ingredients
           if ENV["ONLY"]
             elements_with_ingredients = elements_with_ingredients.select { |d| d[:name].in? ENV["ONLY"].split(",") }
           end
+
           # eager load all elements that have ingredients defined but no ingredient records yet.
           all_elements = Alchemy::Element
             .named(elements_with_ingredients.map { |d| d[:name] })
             .includes(contents: :essence)
```

Comment:
Maybe we skip eager loading for Rails 6.1 then? This is not ideal, but at least it works.

Reply:
What about the suggestion from [here](https://github.com/AlchemyCMS/alchemy_cms/issues/2174#issuecomment-898427142)?

Reply:
It wonders me why this should work, because as [I stated in my comment](https://github.com/AlchemyCMS/alchemy_cms/issues/2174#issuecomment-1096686393) the argument to `where` is the `table_name` not the relation name. https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-where-label-Joins

Reply:
Yes, I knew that. I wanted to investigate why, but I had no time to work on it yet.

Reply:
I've found [this](https://guides.rubyonrails.org/v6.1/active_record_querying.html#using-array-hash-of-named-associations) in the Rails docs:

> Active Record lets you use the names of the [associations](https://guides.rubyonrails.org/v6.1/association_basics.html) defined on the model as a shortcut for specifying JOIN clauses for those associations when using the joins method.